NEWS.md

```diff
@@ -37,6 +37,10 @@ CHANGES
 
 NEW FUNCTIONS
 
+* `unstandardise()` now works for `center(x)`
+
+* `unnormalize()` now works for `change_scale(x)`
```

Comment:
These NEWS entries shouldn't be under `NEW FUNCTIONS` section.

If it doesn't exist, you can create a new `MAJOR CHANGES` section.